.github/workflows/dummy_po_root.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
We probably should change issuer/subject names so that they say that this certificate should not be used in production systems, in case somebody tries to use artifacts produced by CI.

Reply:
Done